packages/dev/core/src/Misc/sceneSerializer.ts

```diff
@@ -156,11 +156,19 @@ export class SceneSerializer {
         serializationObject.useRightHandedSystem = scene.useRightHandedSystem;
 
         // Fog
-        if (scene.fogMode && scene.fogMode !== 0) {
+        if (scene.fogMode) {

```

Comment:
Fog mode 0 is a valid value. Same for `fogStart` / `fogEnd` / `fogDensity`.

Maybe you want to test against `undefined` instead?

Reply:
This is what happens when you code and play wow simultaneously ;D